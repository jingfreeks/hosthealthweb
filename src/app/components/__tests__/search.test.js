import { render } from "@testing-library/react";
import Search from "../search";

describe("Search Component", () => {
  it("renders Search Component to make snapshots", () => {
    const { container } = render(<Search />);
    expect(container).toMatchSnapshot();
  });
});
