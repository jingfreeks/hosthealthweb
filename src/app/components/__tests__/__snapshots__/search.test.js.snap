// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Component renders Search Component to make snapshots 1`] = `
<div>
  <form
    class="w-50 flex justify-center md:justify-between"
  >
    <input
      class="bg-white p-2 w-80 text-xl rounded-xl"
      placeholder="Search"
      type="text"
      value=""
    />
    <button
      class="p-2 text-xl rounded-xl bg-slate-300 ml-2 font-bold"
    >
      🔍
    </button>
  </form>
</div>
`;
